frontend/modules/model/model.sagas.ts

```diff
@@ -58,11 +58,13 @@ export function* updateSettings({ modelData: { teamspace, project, modelId }, se
 
 		yield API.editModelSettings(teamspace, modelId, modifiedSettings);
 
-		yield put(
-			TeamspacesActions.updateModelSuccess(
-				teamspace, modelId, { project, model: modelId, name: modifiedSettings.name, code: modifiedSettings.code }
-			)
-		);
+		if (modifiedSettings.name && modifiedSettings.code) {
```

Comment:
does this make sense? not all model settings have a code.  Did you run into some errors?

Reply:
Good catch, I had an issue during setting default legend but I checked how this was implemented in case of default view and I realized that I overcomplicate it, nothing needs to be changed here, I'm pushing update right now.